frontend/ui/widget/pagebrowserwidget.lua

```diff
@@ -1048,8 +1107,27 @@ Tap on a thumbnail to read this page.
 
 Long-press on ≡ to decrease or reset the number of chapter levels shown in the bottom ribbon.
 
-Any multiswipe will close the page browser.]]),
-    })
+Any multiswipe will close the page browser.]])
+        elseif Device:hasKeyboard() then
+            text = _([[
+Use settings in this menu to change the number of columns and rows to be shown, whether to display page numbers and, different chapter-levels in bottom ribbon.
+
+Press Shift+Up to move up by one row, or LPgBack/RPgBack to move one screen.
```

Comment:
Are those the going user friendly(ish) terms as will also be used in the user guide by @offset-torque?

Reply:
Ok now I see. So we are still playing the "cryptic buttons" game. Those buttons have much more clearer official names and I will use these names in our guide of course. I can't say what to do here but I suggest avoiding this camel-case-variable-like naming **especially in a help text**. When I searched for "LPgBack" to check, Google brought me **"liquefied petroleum gas back pressure check valves"** so that's not even a searchable word.

![kindles](https://github.com/user-attachments/assets/030ffc07-0db4-4633-88f8-9647c37e2e12)

Reply:
Agreed. I figured maybe they came from the Kindle manual.